projects/extension/src/containers/Options.tsx

```diff
@@ -184,6 +184,12 @@ const Options: React.FunctionComponent = () => {
   }, [poolingLogs])
 
   useEffect(() => {
+    chrome.storage.local.get(["crashError"], (res) => {
+      if (res.crashError) {
+        setValue(2)
```

Comment:
What's "value"? Can we have a better name, or if not at the very least a comment?